webui/react/src/components/TaskList.tsx

```diff
@@ -264,6 +305,7 @@ const TaskList: React.FC<Props> = ({ workspace }: Props) => {
       <TableFilterDropdown
         {...filterProps}
         multiple
+        searchable
```

Comment:
question: I don't have a problem with the state filter being searchable, but it seems outside the scope of this PR. Any reason this was added?